templates/UmbracoExtension/README.txt

```diff
@@ -0,0 +1,30 @@
+      _       _              _                                                          
+     | |     | |            | |                                                         
+   __| | ___ | |_ _ __   ___| |_   _ __   _____      __                                 
+  / _` |/ _ \| __| '_ \ / _ \ __| | '_ \ / _ \ \ /\ / /                                 
+ | (_| | (_) | |_| | | |  __/ |_  | | | |  __/\ V  V /                                  
+  \__,_|\___/ \__|_| |_|\___|\__| |_| |_|\___| \_/\_/   _                 _             
+                 | |                                   | |               (_)            
+  _   _ _ __ ___ | |__  _ __ __ _  ___ ___     _____  _| |_ ___ _ __  ___ _  ___  _ __  
+ | | | | '_ ` _ \| '_ \| '__/ _` |/ __/ _ \   / _ \ \/ / __/ _ \ '_ \/ __| |/ _ \| '_ \ 
+ | |_| | | | | | | |_) | | | (_| | (_| (_) | |  __/>  <| ||  __/ | | \__ \ | (_) | | | |
+  \__,_|_| |_| |_|_.__/|_|  \__,_|\___\___/   \___/_/\_\\__\___|_| |_|___/_|\___/|_| |_|
+                                                                                        
+
+== Requirements ==
+* Node LTS Version 20.17.0+
+
+== Steps ==
+* Open `\Client` folder in Visual Studio Code
```

Comment:
This step seems completely unnessecary as nothing in the following steps are done inside VS Code 🙂

Reply:
Yep can all be done in the terminal, I will reword it and then suggest for doing actual dev work to use VSCode for a better experience.